x/group/keeper/keeper_test.go

```diff
@@ -452,7 +455,10 @@ func (s *TestSuite) TestTallyProposalsAtVPEnd() {
 	s.Require().NoError(err)
 
 	s.Require().NoError(s.groupKeeper.TallyProposalsAtVPEnd(ctx))
-	s.NotPanics(func() { module.EndBlocker(ctx, s.groupKeeper) })
+	s.NotPanics(func() {
+		err := module.EndBlocker(ctx, s.groupKeeper)
+		s.Require().NoError(err)
```

Comment:
Can you check if endblocker does not panic at all anymore? Maybe it was changed in a refactor.
If that's the case, we can simply that to do a just an error check.